playbooks/deploy-ocp-hybrid-multinode.yml

```diff
@@ -233,6 +236,37 @@
       ansible.builtin.import_role:
         name: redhatci.ocp.generate_manifests
 
+    - name: Get stats for each file
+      ansible.builtin.stat:
+        path: "{{ item }}"
+      loop: "{{ extra_manifests }}"
+      register: file_stats
```

Comment:
I think a better name here will be more convenient.

Reply:
those are extra manifest what should I call them ?

Reply:
extra_manifests_stats
just for better convenience, when I reviewed the middle part it wasnt to clear.

Reply:
ok for the register -- OK got it give me a sec

Reply:
done